src/components/CandidateStatus.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import { votes } from '../axios/auth';
```

Comment:
요 부분 폴더 이름이랑 export name을 좀 바꿔서
```suggestion
import requestSenders from '../axios/auth';
```

...
해서

```
requestSenders.voteForId(id)
```

이런식으로 사용해보는 건 어떨까요?
코드가 길어지거나 처음 코드를 보는 사람 입장에서는
votes()라는 이름보단 requestSenders.voteForId(id)가 더 직관적일 수 있을 것 같아요.

Reply:
그러면  auth가 아니라 파일명이 requestSenders가 되는걸까요??